packages/calcite-components/src/components/block/block.scss

```diff
@@ -93,9 +102,18 @@ calcite-handle {
 }
 
 .icon {
```

Comment:
remove?

Reply:
icon is still used with deprecated items